src/networking/SensorMQTT.cpp

```diff
@@ -115,16 +115,24 @@ bool SensorMQTT::subscribeToTopics() {
 }
 
 void SensorMQTT::publishBootEvent(bool firstBoot, int connDur) {
+  // Read RSSI and publish for every reset...
+  float rssiRD;
+  rssiRD = WiFi.RSSI();
+  if (rssiRD < -99)
+  {
+    rssiRD = -99;
+  }
```

Comment:
Would prefer this in a method of it's own that returns the float, but for now it's fine @TomFrames